cmd/nerdctl/ps.go

```diff
@@ -291,6 +306,57 @@ func getPrintableContainerName(containerLabels map[string]string) string {
 	return ""
 }
 
+type containerVolume struct {
```

Comment:
The code change in this PR may be useful when we want to impl the `container ls --filter`. I suggest moving some of the code to a single file to make reusing the code easier